lms/static/scripts/frontend_apps/components/GroupConfigSelector.js

```diff
@@ -88,13 +171,17 @@ export default function GroupConfigSelector({
           path: listGroupSetsAPI.path,
         })
       );
-      setGroupSets(groupSets);
+      if (groupSets && groupSets.length === 0) {
+        setFetchError(new GroupListEmptyError());
+      } else {
+        setGroupSets(groupSets);
+      }
     } catch (error) {
       setFetchError(error);
     }
   }, [authToken, listGroupSetsAPI]);
 
-  const onErrorCancel = () => {
+  const onErrorCancel = useCallback(() => {
```

Comment:
`useCallback` here? Yes, no? Pointless?

Reply:
Yes, this is fine. The UI is pretty simple so the impact will be small, but it won't hurt.